node/src/blockchain/blockchain_interface.rs

```diff
@@ -178,19 +185,19 @@ impl BlockchainInterfaceNull {
         }
     }
 
-    fn look_out_am_null<O, E>(&self, operation: &str) -> Result<O, E>
+    fn interface_uninitialized<O, E>(&self, operation: &str) -> Result<O, E>
     where
         E: BlockchainInterfaceUninitializedError,
     {
-        self.log_null_translates_as_bi_uninitialized(operation);
-        Err(E::bi_uninitialized_error())
+        self.log_interface_uninitialized(operation);
+        Err(E::interface_uninitialized_error())
     }
 
-    fn log_null_translates_as_bi_uninitialized(&self, operation: &str) {
+    fn log_interface_uninitialized(&self, operation: &str) {
         error!(
             self.logger,
-            "Trying to {} with uninitialized blockchain interface. The blockchain \
-        service URL was not set.",
+            "Trying to {} with uninitialized blockchain interface. Parameter blockchain-service-url \
```

Comment:
"Failed to {}" would be better than "Trying to {}", because the latter isn't specific about what happened as a result.